schema/ClaimsAttributeObject.md

```diff
@@ -0,0 +1,36 @@
+# Attribute Object
+
+## Summary
+An Attribute Object contains attribute information the TDF3 system uses to enforce access control. Attributes are used in both the [PolicyObject](PolicyObject.md) to define the attributes that a user "needs" to gain access in an ABAC sense, and in the [ClaimsObject](ClaimsObject.md) to assert the attributes that a user "has" to satisfy the ABAC needs.
+
+The _attribute_ field must be both unique and immutable as it is the reference id for the attribute. All of the other fields are mutable. The attribute string contains three pieces of information - the authority namespace (https://example.com), the attribute name (classification), and the attribute value (topsecret).
+
+The public key is used to wrap the object key or key splits on TDF3 file creation. On decrypt, the kasUrl defines where this key or key split can be rewrapped. For policies that do not include attributes these values are extracted from a _default_ attribute. Every [ClaimsObject](ClaimsObject.md) to a user who may write attribute-free policies should include one and only one _default_ attribute.
+
+The AttributeObject does not define how the attribute will be used. The KAS uses attribute policies from the cognizant authority to make its policy decisions. Clients writing policies should use best available information from their organizations to select which AttributeObjects to include to protect the policy.  
+
+## Version
+
+The current schema version is `1.1.0`.
```

Comment:
I know there was some discussion about whether we need a schema version or whether it means anything - where is this schema version tracked and how do people know when to update it, and if we are going to track the current version here, do they know they need to update this doc when they do that?